fetch.bs

```diff
@@ -1758,6 +1757,46 @@ of which is one of `<code>DPR</code>`, `<code>Save-Data</code>`, `<code>Viewport
 `<code>Width</code>`.
 
 
+<h3 id=accept-ch-cache>Accept-CH cache</h3>
+
+<p>An <dfn id=concept-accept-ch-cache>Accept-CH cache</dfn> is owned by the user agent and consists
+ of a collection of entries where each entry has these fields:
+
+<ul class=brief>
+ <li><dfn id=concept-accept-ch-cache-origin for=accept-ch-cache>origin</dfn> (an <a for=/>origin</a>)
+ <li><dfn id=concept-accept-ch-cache-max-age for=accept-ch-cache>max-age</dfn> (a number of seconds)
+ <li><dfn id=concept-accept-ch-cache-client-hints-list for=accept-ch-cache>client-hints list</dfn>
+</ul>
+
+<p>Entries must be removed after the seconds specified in the
```

Comment:
Do we need any extra clauses to ensure that this cache is purged when user clears site-data or similar?

Reply:
Probably

Reply:
@mikewest what's the best way to hook this to clear-site-data or similar UI concepts (e.g. user cleared their cookies)? Looking at https://w3c.github.io/webappsec-clear-site-data/#clear-dom it seems to enumerate all the relevant things that need to be cleared? Should the ACHL cache do the same?